src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -89,33 +144,69 @@ public static void addInventoryListener(InventoryListener l) {
     private JButton validateButton;
     private JScrollPane scrollPaneInventory;
     private JTable jTableInventoryRow;
+    private List<MedicalInventoryRow> inventoryRowList;
+    private List<MedicalInventoryRow> inventoryRowSearchList;
     private String[] pColums = { MessageBundle.getMessage("angal.common.code.txt"),
             MessageBundle.getMessage("angal.inventory.medical.col"),
+            MessageBundle.getMessage("angal.inventory.lotcode.col"),
```

Comment:
Please review code in other table definitions; column titles should be in uppercase; use `toUpperCase()`